main.py

```diff
@@ -77,7 +77,7 @@ def create_app():
     return app
 
 if DEV_MODE:
```

Comment:
Flask convention is to set a env variable:
https://flask.palletsprojects.com/en/1.1.x/config/#environment-and-debug-features

This also gives you secret hacker debugger tools.

Reply:
For now, testing is done with `python3 main.py --dev` instead of using Flask. Will implement the flask convention a little later :)